File: 7KYU/esrever esreveR!/solution.py
def esrever(str):
  if len(str) == 0:
      return ""
  but_last = str[:-1]
  reversed_string = ""
  for i in but_last.split(' '):
    reversed_string = i[::-1] + " " + reversed_string
  return reversed_string[:-1] + str[-1]

File: 7KYU/esrever esreveR!/solution.js
function reverse(s){
    return s.split('').reverse().join('');
  }
  
  function esrever(s) {
    return (s == '')
      ? ''
      : (s.slice(0, -1).split(' ')
                       .reverse()
                       .map(m => reverse(m))
                       .join(' '))  + s[s.length-1];
}
  